package.json

```diff
@@ -21,7 +21,8 @@
     "lint": "tslint -p .",
     "gcp-build": "cd client && tsc -p .",
     "deploy": "gcloud app deploy",
-    "pretty": "prettier --write \"./**/*.{tsx,css,js,json}\""
+    "pretty": "prettier --write \"./**/*.{tsx,css,js,json}\"",
+    "serverOnly": "cd server && npm start"
```

Comment:
Okay I will do npm i after this gets merged

Reply:
It runs `npm i` every time you run `npm run develop` automatically so don't worry about that.